uimaj-core/pom.xml

```diff
@@ -78,6 +78,11 @@
 			<artifactId>assertj-core</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```

Comment:
This dependency should have a <scope> of test, right?

Reply:
I only added test code - so yes